### ISSUE TITLE
Improve files error load display

### ISSUE THREAD

**Issue by Mikescops** [nextcloud/photos#508]
When files fail loading we should display a nice error. 

For now we keep the preview image when there is an error.

In the future we would like to fail nicely with a "broken image" icon or fall back to the image alt.

This will be useful for videos when previews are not enabled.

![image](https://user-images.githubusercontent.com/4266283/96896625-0531e680-148e-11eb-820b-d7edb1090f47.png)

**Comment by jancborchardt** [comment]
I’d say showing the video file type icon and the filename below would be a good combination. :) Then it’s quick to see visually (also in a list mixed with photos), and not only by the file extension.

**Comment by Mikescops** [comment]
Yes I was thinking about this too, but the image above was just a quick try x)
